packages/in_app_purchase/in_app_purchase_android/android/src/main/java/io/flutter/plugins/inapppurchase/MethodCallHandlerImpl.java

```diff
@@ -55,7 +51,9 @@
   private final Context applicationContext;
   private final MethodChannel methodChannel;
 
-  private HashMap<String, SkuDetails> cachedSkus = new HashMap<>();
+  // TODO(stuartmorgan): Migrate this code. See TODO on querySkuDetailsAsync.
+  @SuppressWarnings("deprecation")
+  private HashMap<String, com.android.billingclient.api.SkuDetails> cachedSkus = new HashMap<>();
```

Comment:
Having to inline namespaces everywhere is the most annoying part of the change, but having `-Werror` is more important IMO. And all of this will go away when the code is migrated to v5.

Reply:
v5 of what?